tests/test_download_desktop.py

```
@@ -2307,5 +2283,584 @@ def test_manage_prerelease_tracking_files_filenotfound(downloader, tmp_path):
+
+    config = {"DOWNLOAD_DIR": str(tmp_path / "downloads")}
+    dl = MeshtasticDesktopDownloader(
+        config, CacheManager(cache_dir=str(tmp_path / "cache"))
+    )
+    dl.migrate_split_layout()
+
+    assert (tmp_path / "downloads" / "app" / "prerelease" / "v2.7.20-open.1").exists()
+
+
+def test_get_known_2714_prerelease_mismatch_tags_sorts_non_matching():
+    config = {"DOWNLOAD_DIR": "/tmp/test"}
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)